evadb/third_party/databases/github/github_handler.py

```diff
@@ -0,0 +1,147 @@
+# coding=utf-8
+# Copyright 2018-2023 EvaDB
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import github
+import pandas as pd
+
+from evadb.third_party.databases.github.table_column_info import STARGAZERS_COLUMNS
+from evadb.third_party.databases.types import (
+    DBHandler,
+    DBHandlerResponse,
+    DBHandlerStatus,
+)
+
+
+class GithubHandler(DBHandler):
```

Comment:
@kaushikravichandran @xzdandy Do we want to merge #1033 PR first?

Reply:
We'd like to introduce `SELECT`, `WRITE`, `CREATE`, `DROP` for the non-sql data source integration. And use them in the native_storage_engine.py. Some efforts are needed to refactor the pull request. To unblock the efforts for optimization in a short time period, I choose to do an implementation for Github data source first.  This PR includes the `SELECT` part, which can be a reference for the rest interfaces.